src/Command/NewCommand.php

```diff
@@ -66,7 +66,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
     $this->initializeGitRepository($dir);
 
     $output->writeln('');
-    $output->writeln("<info>New 💧Drupal project created in $dir. 🎉");
+    $output->writeln("<info>New 💧 Drupal project created in $dir. 🎉");
```

Comment:
That works, but it seems we also have an extra white space after "New" FYI
<img width="340" alt="Screenshot_20210225_154716" src="https://user-images.githubusercontent.com/293478/109170067-c041f780-7780-11eb-8f58-d618d4c9625e.png">

Reply:
It's not extra. Just 1 space.